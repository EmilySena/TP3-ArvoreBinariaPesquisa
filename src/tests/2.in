i kalamin
e 2 kalamin kalamin
s kalamin aarinhun
i khulksna
i lobstins
i minu
i prixah
i zhitun
d 6 6 5 4 3 2 1
s zhitun brinh
s prixah cloxin
s minu draxt
s lobstins frix
e 4 frix draxt cloxin brinh
s khulksna hiulx
e 4 frix draxt cloxin brinh
s aarinhun zion
e 4 frix draxt cloxin brinh
d 2 6 1
