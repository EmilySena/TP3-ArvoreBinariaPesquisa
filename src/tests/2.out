1 1
zhitun prixah minu lobstins khulksna aarinhun
6 5 4 3
5 4 3 2
4 3 2 1
zion brinh
